samples/embedded-auth-with-sdk/src/test/java/pages/Page.java

```diff
@@ -151,6 +151,30 @@ public String fetchMagicLinkFromEmail(String emailContent) {
         return matcher.find() ? matcher.group(1) : null;
     }
 
+    public String fetchOktaVerifyMagicLinkFromEmail(String emailContent) {
+        Pattern pattern = Pattern.compile("\\\"push-verify-activation-link\\\" href=\\\"(.*?)\\\"");
+        Matcher matcher = pattern.matcher(emailContent);
+        return matcher.find() ? matcher.group(1) : null;
+    }
+
+    public String fetchMagicLinkFromSMS() {
```

Comment:
Rename method to `fetchLinkFromSMS()` 
There's no magic link here.
Remove all references to magic.

Reply:
renamed the references